README.md

```diff
@@ -0,0 +1,76 @@
+# HackWeek: Learn | Combine | Share
+
+HackWeek is a SUSE way of [Hackathon](https://en.wikipedia.org/wiki/Hackathon),
+which is usually one full week long. This time it's actually six days,
+overlaping one day with [openSUSE conference](https://events.opensuse.org/)
+in Nürnberg.
+
+## Who | When | How
+
+* Anyone at SUSE can participate [^1], in team or alone, what works for you
+* 24th June - 1st July 2016
+* Depending on your local conditions, but basically you just come to work and
+  start hacking
+
+## HackWeek Theme
+
+HackWeek #14 is a bit special, this time we have several suggested themes:
+
+* [NFV](https://en.wikipedia.org/wiki/Network_function_virtualization) - Network
+  function virtualization
+* Cloud
+* Storage
+* Orchestration
+* ... and obviously their combinations or in combination with other current
+  SUSE solutions
+
+## Current Projects
+
+All current projects can be found at
+[HackWeek Pages](https://hackweek.suse.com/14/projects)
+but you are kindly asked to create new projects there as well.
+
+## Create New Project
+
+You have a great idea what could be done, but you just have two hands? You have
+a plenty of great ideas, but you can just work on two of them in parallel? You
+are looking for contributors? Anything else? Just go to the
+[HackWeek Page](https://hackweek.suse.com/14/projects/new) and file it there.
+
```

Comment:
This paragraph gives the impression that you should only create the project if you need help or somebody else to do it. But you should also create the project even if you plan to work on it alone.